libs/ledger-live-common/src/wallet-api/react.ts

```diff
@@ -828,6 +836,24 @@ export interface RecentlyUsed {
 
 export type RecentlyUsedManifest = AppManifest & { usedAt?: Date };
 
+function calculateTimeDiff(usedAt: string) {
+  const now = new Date();
+  const date = new Date(usedAt);
+
+  const units = [
+    { translationKey: "months", diff: differenceInMonths(now, date) },
+    { translationKey: "weeks", diff: differenceInWeeks(now, date) },
+    { translationKey: "days", diff: differenceInDays(now, date) },
+    { translationKey: "hours", diff: differenceInHours(now, date) },
+    { translationKey: "minutes", diff: differenceInMinutes(now, date) },
+    { translationKey: "seconds", diff: differenceInSeconds(now, date) },
+  ];
```

Comment:
[Nitpick] FYI date-fns [intervalToDuration](https://date-fns.org/v2.16.1/docs/intervalToDuration) does the same job here in one function call:  



```js 
const durationDifference =  intervalToDuration({start: parsedDate, end: now})

/* gives back 


durationDifference : 
days: 15,
hours: 3,
minutes: 40,
months: 1,
seconds: 59,
years: 6
*/